src/mpol/losses.py

```diff
@@ -267,3 +267,30 @@ def PSD(qs, psd, l):
     loss = torch.sum(psd / expected_PSD)
 
     return loss
+
+
+def TSV(sky_cube):
+    r"""
+    Calculate the total square variation (TSV) loss in the image dimension (R.A. and DEC). Following the definition in `EHT-IV 2019 <https://ui.adsabs.harvard.edu/abs/2019ApJ...875L...4E/abstract>`_ Promotes the image to be edge smoothed which may be a better reoresentation of the truth image `K. Kuramochi et al 2018 <https://ui.adsabs.harvard.edu/abs/2018ApJ...858...56K/abstract>`_.
+
+    Args:
+        sky_cube (any 3D tensor): the image cube array :math:`I_{lmv}`, where :math:`l` is R.A., :math:`m` is DEC, and :math:`v` is the channel (velocity or frequency) dimension. Should be in sky format representation.
```

Comment:
This should be more specific about how the sky cube is packed. Specifically, frequency in ndim=1, m in ndim=2, and l in ndim=3.